fvtest/compilertriltest/CompareTest.cpp

```diff
@@ -52,11 +52,16 @@ TEST_P(Int32Compare, UsingConst) {
     auto param = TRTest::to_struct(GetParam());
 
     char inputTrees[120] = {0};
-    std::snprintf(inputTrees, 120, "(method return=Int32 (block (ireturn (%s (iconst %d) (iconst %d)))))", param.opcode.c_str(), param.lhs, param.rhs);
+    std::snprintf(inputTrees, 120,
+       "(method return=Int32 "
+         "(block "
+           "(ireturn "
+             "(%s "
+               "(iconst %d) "
+               "(iconst %d)))))",
+       param.opcode.c_str(), param.lhs, param.rhs);
     auto trees = parseString(inputTrees);
 
-    ASSERT_NOTNULL(trees);
```

Comment:
Why is this being removed? I think it's still needed.

Reply:
That was not an intended change.  Updated the file.